### PR TITLE
Update iTunes module for iOS 11 new podcast's rss specs

### PR DESCRIPTION
**Pull request by razyalov**
Update iTunes module for iOS 11 new podcast's rss specs

Apple released new guidlines for the itunes RSS spec for podcasts: http://podcasts.apple.com/resources/spec/ApplePodcastsSpecUpdatesiOS11.pdf

The specific technical changes are the addition of the following attributes:

Channel level: The following was added:

itunes:type :: type of podcast 'episodic' (default)  or 'serial'.

Item level: The following was added:

itunes:season :: a non-zero integer number for a season identifier
itunes:episode :: a non-zero integer episode number within the season
itunes:episodeType :: one of 'full' (default) 'bonus' or 'trailer'
itunes:encoded :: show notes with basic HTML formatting

All new values are optional.

### CODE REVIEW

File: rome-modules/src/test/resources/itunes/leshow.xml [hunk 1]
```diff
@@ -65,7 +66,11 @@
 	
 	    <itunes:duration>46:34</itunes:duration>
 	
-    <itunes:keywords></itunes:keywords>  
+    <itunes:keywords></itunes:keywords>
+    <itunes:encoded><![CDATA[show notes with <b>bold</b> text.]]></itunes:encoded>
```
Comment (mishako) [inline rome-modules/src/test/resources/itunes/leshow.xml]: The pdf says it's `<content:encoded>`. I don't know what they mean by the `content` namespace, but isn't it suspicious that it's different from all other tags?

**Comment by razyalov** [comment]
good catch. I agree. I’ll do some more digging.

Raz

File: rome-modules/src/test/resources/itunes/leshow.xml (hunk 1 above)
Comment (mishako) [reply rome-modules/src/test/resources/itunes/leshow.xml]: Looks like we already have this tag in the `content` module: https://github.com/rometools/rome/blob/master/rome-modules/src/main/java/com/rometools/modules/content/ContentModuleImpl.java#L43

**Comment by razyalov** [comment]
Ok, I guess this is not a new element. content:encoded exists as part of the rss standards from what I can see.
https://www.w3.org/wiki/RssContent <https://www.w3.org/wiki/RssContent>

So I’ll just remove it from my change.

Raz

**Comment by mishako** [comment]
:+1: I'll wait a couple of days before merging in case others want to review.

**Comment by razyalov** [comment]
awesome.

Thanks,
Raz

**Comment by razyalov** [comment]
Thanks.

Any idea when the next release is coming out with these changes?

Raz

**Pull request merged**